WPThemeReview/ruleset.xml

```diff
@@ -150,4 +150,10 @@
 	<!-- Themes should never touch the timezone. -->
 	<rule ref="WordPress.WP.TimezoneChange"/>
 
+	<!-- Exclude PrefixAllGlobals Sniff from WPCS and use the one from this ruleset -->
```

Comment:
Makes much more sense. I wasn't sure if I need this because I want to have its functionality, but it will be included via the new sniff (only slightly changed for themes). :+1: